JSDoc type import parsing broken in JSDoc typedefs

<!-- 🚨 STOP 🚨 𝗦𝗧𝗢𝗣 🚨 𝑺𝑻𝑶𝑷 🚨

Half of all issues filed here are duplicates, answered in the FAQ, or not appropriate for the bug tracker. Even if you think you've found a *bug*, please read the FAQ first, especially the Common "Bugs" That Aren't Bugs section!

Please help us by doing the following steps before logging an issue:
  * Search: https://github.com/Microsoft/TypeScript/search?type=Issues
  * Read the FAQ: https://github.com/Microsoft/TypeScript/wiki/FAQ

Please fill in the *entire* template below.
-->

<!-- Please try to reproduce the issue with `typescript@next`. It may have already been fixed. -->
**TypeScript Version:**  3.2.0-dev.20181024

<!-- Search terms you tried before logging this (so others can find this issue more easily) -->
**Search Terms:** jsdoc import parsing

**Code**

```js
/**
 * @typedef {Object} options
 * @property {import("./module").MyClass} option
 */
```

**Expected behavior:**

The module `./module` can be resolved and type `MyClass` is recognized. This used to work before commit ff6f94791feeb9c2b22ff1ada41c173a61f347fb.

**Actual behavior:**

Since commit ff6f94791feeb9c2b22ff1ada41c173a61f347fb, this generates the following error message:
```
error TS2307: Cannot find module './module.js'.
```

--- a/tests/baselines/reference/jsdocImportTypeResolution.symbols
+++ b/tests/baselines/reference/jsdocImportTypeResolution.symbols
@@ -1,0 +1,14 @@
+=== tests/cases/compiler/module.js ===
+export class MyClass {
+>MyClass : Symbol(MyClass, Decl(module.js, 0, 0))
+}
+
+=== tests/cases/compiler/usage.js ===
+/**
+ * @typedef {Object} options
+ * @property {import("./module").MyClass} option
+ */
+/** @type {options} */
+let v;
+>v : Symbol(v, Decl(usage.js, 5, 3))
+

--- a/tests/baselines/reference/jsdocImportTypeResolution.types
+++ b/tests/baselines/reference/jsdocImportTypeResolution.types
@@ -1,0 +1,14 @@
+=== tests/cases/compiler/module.js ===
+export class MyClass {
+>MyClass : MyClass
+}
+
+=== tests/cases/compiler/usage.js ===
+/**
+ * @typedef {Object} options
+ * @property {import("./module").MyClass} option
+ */
+/** @type {options} */
+let v;
+>v : options
+

--- a/tests/cases/compiler/jsdocImportTypeResolution.ts
+++ b/tests/cases/compiler/jsdocImportTypeResolution.ts
@@ -1,0 +1,14 @@
+// @allowJs: true
+// @noEmit: true
+// @checkJs: true
+// @filename: module.js
+export class MyClass {
+}
+
+// @filename: usage.js
+/**
+ * @typedef {Object} options
+ * @property {import("./module").MyClass} option
+ */
+/** @type {options} */
+let v;

